### PR TITLE
Helper function to create a container of a widget with label to the left

### PR DESCRIPTION
**Pull request by phil-brad**

### CODE REVIEW

File: pyqttoolkit/views/labelled.py [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program; if not, write to the Free Software Foundation,
+# Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301  USA
+from PyQt5.Qt import QWidget, QHBoxLayout, QLabel
+
+
+def make_labelled(parent, label: str, widget: QWidget):
```
Comment (mtinning) [inline pyqttoolkit/views/labelled.py:20]: If you add lots of rows of these widgets, they won't line up on the horizontal axis. Assuming this is intentional, that's fine.

File: pyqttoolkit/views/labelled.py [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+    """Put a widget into a container with a label to the left of it. Like a single row of a `QFormLayout`"""
+    container = QWidget(parent)
+    label = QLabel(label)
+    layout = QHBoxLayout(container)
+    layout.setSpacing(0)
+    layout.addWidget(label)
+    layout.addWidget(widget)
+    layout.setContentsMargins(0, 0, 0, 0)
+    container.setLayout(layout)
```
Comment (mtinning) [inline pyqttoolkit/views/labelled.py]: This line won't do anything as the constructor of QHBoxLayout will do this for you

File: pyqttoolkit/views/labelled.py [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+# You should have received a copy of the GNU General Public License
+# along with this program; if not, write to the Free Software Foundation,
+# Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301  USA
+from PyQt5.Qt import QWidget, QHBoxLayout, QLabel
+
+
+def make_labelled(parent, label: str, widget: QWidget):
+    """Put a widget into a container with a label to the left of it. Like a single row of a `QFormLayout`"""
+    container = QWidget(parent)
+    label = QLabel(label)
```
Comment (mtinning) [inline pyqttoolkit/views/labelled.py]: Prefer not to redefine variables to avoid confusion - this could be called `label_widget`

**Review by mtinning — CHANGES_REQUESTED** [review]
Few comments, probably best to address the variable renaming one but take or leave the others

File: pyqttoolkit/views/labelled.py (hunk 1 above)
Comment (phil-brad) [reply pyqttoolkit/views/labelled.py:20]: if one is adding loads of consecutive rows, they should use a `QFormLayout`. This is really about fitting one of these amongst other widgets that don't conform to the form layout

Comment (phil-brad) [reply pyqttoolkit/views/labelled.py:20]: if fact I have added two of these consecutively and they did seem to line up. Probably due to an implicit size policy?

Comment (mtinning) [reply pyqttoolkit/views/labelled.py:20]: Perhaps - I guess my point is they won't necessarily line up. If this isn't the goal of this then that's fine though

**Review by mtinning — APPROVED** [review]
:shipit:

**Pull request merged**